doc/apimanager-reference.md

```diff
@@ -7,55 +7,777 @@ One APIManager custom resource per project is allowed.
+  * [APIManagerSpec](#apimanagerspec)
+  * [ApicastSpec](#apicastspec)
+  * [ApicastProductionSpec](#apicastproductionspec)
+  * [ApicastStagingSpec](#apicaststagingspec)
+  * [CustomPolicySpec](#custompolicyspec)
+  * [CustomPolicySecret](#custompolicysecret)
+  * [BackendSpec](#backendspec)
+  * [BackendRedisPersistentVolumeClaimSpec](#backendredispersistentvolumeclaimspec)
+  * [BackendListenerSpec](#backendlistenerspec)
+  * [BackendWorkerSpec](#backendworkerspec)
+  * [BackendCronSpec](#backendcronspec)
+  * [SystemSpec](#systemspec)
+  * [SystemRedisPersistentVolumeClaimSpec](#systemredispersistentvolumeclaimspec)
+  * [FileStorageSpec](#filestoragespec)
+  * [SystemPVCSpec](#systempvcspec)
+  * [SystemS3Spec](#systems3spec)
+  * [DeprecatedSystemS3Spec](#deprecatedsystems3spec)
+  * [DatabaseSpec](#databasespec)
+  * [MySQLSpec](#mysqlspec)
+  * [SystemMySQLPVCSpec](#systemmysqlpvcspec)
+  * [PostgreSQLSpec](#postgresqlspec)
+  * [SystemPostgreSQLPVCSpec](#systempostgresqlpvcspec)
+  * [SystemAppSpec](#systemappspec)
+  * [SystemSidekiqSpec](#systemsidekiqspec)
+  * [SystemSphinxSpec](#systemsphinxspec)
+  * [ZyncSpec](#zyncspec)
+  * [ZyncAppSpec](#zyncappspec)
+  * [ZyncQueSpec](#zyncquespec)
+  * [ExternalComponentsSpec](#externalcomponentsspec)
+  * [PodDisruptionBudgetSpec](#poddisruptionbudgetspec)
+  * [MonitoringSpec](#monitoringspec)
+  * [APIManagerStatus](#apimanagerstatus)
+    * [ConditionSpec](#conditionspec)
 * [PersistentVolumeClaimResourcesSpec](#persistentvolumeclaimresourcesspec)
 * [APIManager Secrets](#apimanager-secrets)
-   * [backend-internal-api](#backend-internal-api)
-   * [backend-listener](#backend-listener)
-   * [backend-redis](#backend-redis)
-   * [system-app](#system-app)
-   * [system-database](#system-database)
-   * [system-events-hook](#system-events-hook)
-   * [system-master-apicast](#system-master-apicast)
-   * [system-memcache](#system-memcache)
-   * [system-recaptcha](#system-recaptcha)
-   * [system-redis](#system-redis)
-   * [system-seed](#system-seed)
-   * [zync](#zync)
-   * [fileStorage-S3-credentials-secret](#filestorage-s3-credentials-secret)
-   * [system-smtp](#system-smtp)
+  * [backend-internal-api](#backend-internal-api)
+  * [backend-listener](#backend-listener)
+  * [backend-redis](#backend-redis)
+  * [system-app](#system-app)
+  * [system-database](#system-database)
+  * [system-events-hook](#system-events-hook)
+  * [system-master-apicast](#system-master-apicast)
+  * [system-memcache](#system-memcache)
+  * [system-recaptcha](#system-recaptcha)
+  * [system-redis](#system-redis)
+  * [system-seed](#system-seed)
+  * [zync](#zync)
+  * [fileStorage-S3-credentials-secret](#filestorage-s3-credentials-secret)
+  * [system-smtp](#system-smtp)
+* [Default APIManager components compute resources](#default-apimanager-components-compute-resources)
+
+Generated using [github-markdown-toc](https://github.com/ekalinin/github-markdown-toc)
+
+## APIManager
```

Comment:
What is this change? It seems to be duplicated

Reply:
Hmm, possibly a plugin I wasn't aware I had on my IDE. removed it

Reply:
I still see duplicated content

Reply:
ah sorry, It's just me adding the metadata section to the toc